lab_gurpreet/lib/index.js

```diff
@@ -0,0 +1,10 @@
+var cowsay = require('cowsay');
+const newCowsay = module.exports = function(a){
```

Comment:
It's best practice to give your variables more descriptive names, like 'str' or 'input' instead of 'a' in this case